### PR TITLE
Unit testing

### PR DESCRIPTION
**Pull request by CJPoll**
Lay down groundwork for testing our app

### CODE REVIEW

File: tests/test_example.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 class TestTheTruth(unittest.TestCase):
     def test_truth(self):
-        self.assertTrue(True)
+            self.assertTrue(True)
```
Comment (thebriz24) [inline tests/test_example.py:5]: This I'm sure is correct but why do you do two indents instead of one?

**Review by thebriz24 — COMMENTED** [review]
I don't know bash that well about all I know is if fi. But it looks simple enough.

File: tests/test_example.py (hunk 1 above)
Comment (CJPoll) [reply tests/test_example.py:5]: Not sure how _that_ happened. One sec...

**Comment by CJPoll** [comment]
Closed in favor of #10

**Pull request closed without merging**